compiler/luci/pass/src/FuseBatchNormWithTConvPass.cpp

```diff
@@ -96,10 +96,27 @@ bool fused_batch_norm_with_tconv(luci::CircleAdd *add)
     return false;
 
   auto filter_out_chn = filter->dim(0).value();
-  if (filter_out_chn != scale->dim(0).value())
-    return false;
-  if (filter_out_chn != shift->dim(0).value())
-    return false;
+  // allow scale/shift and bias shape of [N], [1,N], [N,1], [1,1,N], ...
```

Comment:
Since BN works "channel-wise", the last dimension should be N. So, [N, 1] is not allowed.

The last dimension should be N, and the rest of dimensions should be 1. So, the shape of scale and shift should be [N] or [1, 1, 1, N] (scale and shift are limited to rank 4 in the code).

Reply:
Thanks! I'll update this PR :)